src/contracts/pods/EigenPod.sol

```diff
@@ -335,8 +342,9 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
      */
     function verifyWithdrawalCredentials(
         uint64 oracleTimestamp,
+        BeaconChainProofs.StateRootProof calldata stateRootProof,
         uint40[] calldata validatorIndices,
-        BeaconChainProofs.WithdrawalCredentialProof[] calldata withdrawalCredentialProofs,
+        bytes[] calldata withdrawalCredentialProofs,
```

Comment:
seems like you should be deleting the `WithdrawalCredentialProof` struct type? AFAICT we aren't using it any more.

Reply:
should be deleted, check the diffs in BeaconCHainProofs

Reply:
also seems like the input might be better named `validatorFieldsProofs` instead of `withdrawalCredentialProofs`? since that is the semantic meaning -- this would make it clearer that these proofs are for the `validatorFields` input.

Reply:
> should be deleted, check the diffs in BeaconCHainProofs.

ah mb it's because it was "renamed" in the diffs.
makes sense, would still like the parameter name change.